### PR TITLE
Add sequence alias for collect, custom NoSuchElementException message…

### PR DESCRIPTION
**Pull request by stevenheidel**
…, update developers list

### CODE REVIEW

File: src/test/scala/io/getclump/ClumpApiSpec.scala [hunk 1]
```diff
@@ -103,6 +103,21 @@ class ClumpApiSpec extends Spec {
...
+        val clumps = List(Clump.value(1), Clump.value(2))
+        clumpResult(Clump.sequence(clumps)) mustEqual Some(List(1, 2))
+      }
+      "set" in {
+        val clumps = Set(Clump.value(1), Clump.value(2))
+        clumpResult(Clump.sequence(clumps)) mustEqual Some(Set(1, 2))
+      }
+      "seq" in {
+        val clumps = Seq(Clump.value(1), Clump.value(2))
+        clumpResult(Clump.sequence(clumps)) mustEqual Some(Seq(1, 2))
```
Comment (fwbrasil) [inline src/test/scala/io/getclump/ClumpApiSpec.scala]: it seems to be missing a test for the `varargs` overload

`Clump.sequence(clump1, clump2)`

**Comment by stevenheidel** [comment]
@fwbrasil @williamboxhall forgot that I hadn't pushed this yet. Any comments?

**Comment by fwbrasil** [comment]
:+1:

**Comment by williamboxhall** [comment]
:+1:

**Comment by fwbrasil** [comment]
@stevenheidel Could you make the release for this change? Let me know if you need help with it.

**Pull request merged**